### PR TITLE
update javadoc, deploy gwt-editor, etc.

### PR DESCRIPTION
**Pull request by FrankHossfeld**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -188,6 +188,7 @@
         <snapshotRepository>
             <id>ossrh</id>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+<!--            <url>https://repo.vertispan.com/gwt-snapshot/</url>-->
```
Comment (niloc132) [inline pom.xml:191]: i dont think we need a new commented out line

File: gwt-editor-processor/pom.xml [hunk 2]
```diff
@@ -232,6 +234,11 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>flatten-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>${maven.deploy.plugin}</version>
```
Comment (niloc132) [inline gwt-editor-processor/pom.xml:240]: need to specify skip=false here to override parent, right?

File: gwt-editor/pom.xml [hunk 3]
```diff
@@ -71,6 +71,11 @@
           <moduleName>org.gwtproject.editor.Editor</moduleName>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven.deploy.plugin}</version>
+      </plugin>
```
Comment (niloc132) [inline gwt-editor/pom.xml:78]: same here, add skip=false right?

File: gwt-editor-processor/src/main/java/org/gwtproject/editor/processor/model/BeanMethod.java [hunk 4]
```diff
@@ -178,6 +189,12 @@ private static boolean deepCompare(EditorTypes types, TypeMirror source, TypeMir
     return true;
   }
 
-  /** Returns {@code true} if the BeanLikeMethod matches the method. */
+  /**
+   * Returns {@code true} if the BeanLikeMethod matches the method.
+   *
+   * @param types editor type#
```
Comment (niloc132) [inline gwt-editor-processor/src/main/java/org/gwtproject/editor/processor/model/BeanMethod.java:195]: extra `#`

File: pom.xml (hunk 1 above)
Comment (FrankHossfeld) [reply pom.xml:191]: I was not sure, If we may go bay to your repo ... ;-) ... remove it.

File: gwt-editor-processor/pom.xml (hunk 2 above)
Comment (FrankHossfeld) [reply gwt-editor-processor/pom.xml:240]: oh yes!

**Pull request merged**